docs/User-Guide.md

```diff
@@ -453,7 +456,64 @@ Outputs are additionally logged to /opt/intelmq/var/log/intelmqctl
 
 #### Botnet Concept
 
-The botnet represents all currently configured bots. To get an overview which bots are running, use `intelmqctl status`or use IntelMQ Manager.
+The botnet represents all currently configured bots which are explicitly enabled. To get an overview which bots are running, use `intelmqctl status` or use IntelMQ Manager. Set `"enabled": true` in the runtime configuration to add a bot to botnet. By default, bots will be configured as `"enabled": false` therefore, does not belong to the botnet due protection reasons. See [Bots](Bots.md) for more details on configuration.
```

Comment:
grammar: 
" therefore, does not belong to the botnet due protection reasons" 

--> " therefore, such a bot does not belong to the botnet (by default)."